web/src/lib/api/index.ts

```diff
@@ -62,6 +68,18 @@ export default class WoodpeckerClient extends ApiClient {
     return this._post(`/api/repos/${owner}/${repo}/pipelines`, options) as Promise<Pipeline>;
   }
 
+  // Deploy triggers a deployment for an existing pipeline using the
+  // specified target environment.
+  deployPipeline(owner: string, repo: string, number: number, options: DeploymentOptions): Promise<Pipeline> {
```

Comment:
```suggestion
  deployPipeline(owner: string, repo: string, pipelineNumber: number, options: DeploymentOptions): Promise<Pipeline> {
```

Reply:
If I rename this single call, shouldn't all other methods (`getPipeline', ' getPipelineConfig', 'cancelPipeline'  in the API follow?

(https://github.com/woodpecker-ci/woodpecker/blob/a884e13885b9519c6c7bf7f6e41a2f177207b50d/web/src/lib/api/index.ts#L70-L84)

My proposal: Go with the current parameter name and I will provide a PR to remane the parameters in all methods once this PR is merged.

Reply:
Just saw that line, but sure we should rename them all.

Reply:
In this PR?

Reply:
We can do that in a new one.

Reply:
I will provide one, once this is merged.